src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -104,6 +104,11 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
     setShowDeleteEventModal();
   };
 
+  const dropdownVenueList = venue?.rooms?.map((room) => ({
+    name: room.title,
+    template: room.template,
```

Comment:
Can use decomposition to reduce `room.` spam

Reply:
done da7ba06